examples/dreamcast/vmu/vmu_beep/beep.c

```diff
@@ -0,0 +1,195 @@
+
+    pvr_set_bg_color(0.0f, 0.0f, 0.0f);
+
+    for(;;) {
+        dev = maple_enum_type(0, MAPLE_FUNC_CONTROLLER);
+        vmudev = maple_enum_type(0, MAPLE_FUNC_CLOCK);
+
+        while(!dev) {
+            pvr_wait_ready();
+            pvr_scene_begin();
+            pvr_list_begin(PVR_LIST_OP_POLY);
+            pvr_list_begin(PVR_LIST_TR_POLY);
+
+            plx_fcxt_begin(cxt);
+            w.x = 40.0f; w.y = 200.0f; w.z = 10.0f;
+            plx_fcxt_setpos_pnt(cxt, &w);
+            plx_fcxt_draw(cxt, "Please attach a controller!");
+            plx_fcxt_end(cxt);
+
+            pvr_scene_finish();
+
+            dev = maple_enum_type(0, MAPLE_FUNC_CONTROLLER);
+        }
+
+        while(!vmudev) {
+            pvr_wait_ready();
+            pvr_scene_begin();
+            pvr_list_begin(PVR_LIST_OP_POLY);
+            pvr_list_begin(PVR_LIST_TR_POLY);
+
+            plx_fcxt_begin(cxt);
+            w.x = 40.0f; w.y = 200.0f; w.z = 10.0f;
+            plx_fcxt_setpos_pnt(cxt, &w);
+            plx_fcxt_draw(cxt, "Please attach a vmu!");
+            plx_fcxt_end(cxt);
+
+            pvr_scene_finish();
+
+            vmudev = maple_enum_type(0, MAPLE_FUNC_CLOCK);
+        }
+
+        pvr_wait_ready();
+        pvr_scene_begin();
+        pvr_list_begin(PVR_LIST_OP_POLY);
+        pvr_list_begin(PVR_LIST_TR_POLY);
+        plx_fcxt_begin(cxt);
+
+        w.x = 70.0f; w.y = 70.0f; w.z = 10.0f;
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "VMUBeep Test by Quzar");
+
+        w.x += 130; w.y += 120.0f;
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_setsize(cxt, 30.0f);
+        plx_fcxt_draw(cxt, "0x");
+
+        w.x += 48.0f;
+        plx_fcxt_setpos_pnt(cxt, &w);
+
+        count = 0;
+        while(count <= 7) {
+            if(i == count)
+                plx_fcxt_setcolor4f(cxt, 1.0f, 0.9f, 0.9f, 0.0f);
+            else
+                plx_fcxt_setcolor4f(cxt, 1.0f, 1.0f, 1.0f, 1.0f);
+
+            sprintf(s[count], "%x", n[count]);
+            plx_fcxt_draw(cxt, s[count]);
+
+            count++;
+            w.x += 25.0f;
+        }
+
+        plx_fcxt_setsize(cxt, 24.0f);
+        plx_fcxt_setcolor4f(cxt, 1.0f, 1.0f, 1.0f, 1.0f);
+        w.x = 65.0f; w.y += 50.0f;
+
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "Press left/right to switch digits.");
+        w.y += 25.0f;
+
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "Press up/down to change values.");
+        w.y += 25.0f;
+
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "Press A to start vmu beep.");
+        w.y += 25.0f;
+
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "Press B to stop vmu beep.");
+        w.y += 25.0f;
+
+        plx_fcxt_setpos_pnt(cxt, &w);
+        plx_fcxt_draw(cxt, "Press ABXYS to quit.");
+
+        plx_fcxt_end(cxt);
+        pvr_scene_finish();
+
+		/* Store current button states + buttons which have been released. */
```

Comment:
One last one that I just noticed...
```suggestion
        /* Store current button states + buttons which have been released. */
```